init.zsh

```diff
@@ -17,6 +17,45 @@ if ! autoload -Uz is-at-least || ! is-at-least "$min_zsh_version"; then
 fi
 unset min_zsh_version
 
+# zprezto convenience updater
+function zprezto-update {
+   function cannot-fast-forward {
+       local STATUS="$1"
+       [[ "${STATUS}" ]] && printf "%s\n" "${STATUS}"
+       printf "Unable to fast-forward the changes. You can fix this by "
+       printf "running\ncd '%s' and then\n'git pull' " "${ZPREZTODIR}"
+       printf "to manually pull and possibly merge in changes\n"
+   }
+   pushd -q
+   cd -- "${ZPREZTODIR}"
```

Comment:
This would be better as `pushd -q -- "${ZPREZTODIR}"` so cd won't cause chpwd hooks to run (cd also has a similar option, but they can be combined, so it makes sense to)